jenkins/ci.suse.de/ccp.yaml

```diff
@@ -0,0 +1,66 @@
+- job:
+    name: 'ccp-full-periodic'
+    node: cloud-trackupstream
+
+    parameters:
+      - string:
+          name: ccp_repo
```

Comment:
I would remove these parameters except this one (`ccp_repo`). The others are jenkins internal. I don't see why you would customize them for now.

Reply:
Done.